ETWAnalyzer/Commands/DumpCommand.cs

```diff
@@ -360,7 +362,9 @@ class DumpCommand : ArgParser
         "[green]Show process start times relative to the start of process with id 43188[/green]" + Environment.NewLine +
         " ETWAnalyzer -dump Process -fd xx.etl/.json -NewProcess 1 -timefmt s -ZeroTime ProcessStart -ZeroProcessName 43188" + Environment.NewLine +
         "[green]Dump process of all extracted files in current directory ordered by start time instead of grouped by process name. Print time as UTC.[/green]" + Environment.NewLine +
-        " ETWAnalyzer -dump Process -sortby time -timefmt utc" + Environment.NewLine;
+        " ETWAnalyzer -dump Process -sortby time -timefmt utc" + Environment.NewLine +
+        "[green]Dump processes and filter with Parent Process IDs.Substrings (e.g. *x*) are supported.[/green]" + Environment.NewLine +
```

Comment:
`with Parent Process IDs.Substrings (e.g. *x*) `
should be an example with numbers since process ids are numbers like 
`Dump processes and filter by parent process Ids (e.g. -parent 123;*434* ).`

Reply:
resolved